org-formation/650-identity-providers/_tasks.yaml

```diff
@@ -207,3 +207,37 @@ GithubOidcSageBionetworksScicompProvisioner:
   DefaultOrganizationBinding:
     Account: !Ref ScicompAccount
     Region: us-east-1
+
+GithubOidcSageBionetworksSvcCatalogLibrary:
+  Type: update-stacks
+  DependsOn: GithubOidcSageBionetworks
+  Template: github-oidc-provider-access.njk
+  StackName: !Sub ${resourcePrefix}-${appName}-sage-bionetworks-svc-catalog-library
```

Comment:
not a big deal, but i think i prefer the stack name to contain the exact repo name, so i think `!Sub ${resourcePrefix}-${appName}-sage-bionetworks-service-catalog-library` would be better.